cmd/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"github.com/CESARBR/knot-cloud-storage/internal/config"
+	"github.com/CESARBR/knot-cloud-storage/pkg/server"
+
```

Comment:
@joaoaneto This blank line is necessary here?

Reply:
They aren't necessary @esdrasjnr, thanks for noticing.

Reply:
Done.